README.md

```diff
@@ -144,12 +143,23 @@ unzip ~/tmp_fmltc/closure-library_v20200406.zip
 popd
 ```
 
+## Install JDK
+Depending on your OS and distribution there are various ways to install JDK. See https://www.oracle.com/java/technologies/downloads/ for instructions. 
 
 ## Fill in the values in server/env_variables.yaml
 
 1. Replace `<YOUR-PROJECT-ID>` with the Google Cloud Project ID for your project.
 1. Replace `<YOUR-ORIGIN>` with the base URL that will serve the website.
 
+## Set Version
+**Important!** Make sure the current working directory is the fmltc root directory when you run this
+  commands.
+
+This will set the version of fmltc to the commit hash
+
+```
+echo "{ \"version\": \"$(git rev-parse --short HEAD)\" }" > server/app.properties
```

Comment:
Chuckle.  A user can put anything like there.  It doesn't have to be a commit hash, but yes it does need to exist.  If you wanted to write software that provided suitable defaults in the absence of that file,  I would not object to that change.